hw3/01-characters.js

```diff
@@ -0,0 +1,39 @@
+// url for the Thrones API
+const url = "https://thronesapi.com/api/v2/Characters";
+const loader = document.getElementById("loading");
+
+const fetchThrones = async (thronesUrl) => {
+  try {
+    const response = await fetch(Thronesurl);
```

Comment:
This typo causes an error. Lowercase it to match the parameter name.

Thronesurl -> thronesUrl

`const fetchThrones = async (thronesUrl) => {
  try {
    const response = await fetch(thronesurl);`